test/integration/handle-404-test.js

```diff
@@ -53,20 +53,100 @@ test('forward all requests that accept html to app', function (group) {
     })
```

Comment:
line 49 should now be `t.match(response.payload, /<html/, 'response is HTML')` as we we return our custom 404.html page for non-html requests